packages/adyen-salesforce-pwa/lib/components/applePayExpress.jsx

```diff
@@ -9,9 +9,22 @@ import {getAppleButtonConfig, getApplePaymentMethodConfig} from './helpers/apple
 import {AdyenShippingMethodsService} from '../services/shipping-methods'
 
 const ApplePayExpressComponent = (props) => {
-    const {authToken, customerId, locale, site, basket, navigate, onError = []} = props
-
-    const basketId = basket?.basketId
+    const {
+        authToken,
+        customerId,
+        locale,
+        site,
+        basket,
+        navigate,
+        onError = [],
+        currency,
+        isExpressPdp = false,
+        merchantDisplayName = '',
+        product
+    } = props
+    const [shopperBasket, setShopperBasket] = useState(
```

Comment:
shopperBasket is used as a state here but the setShopperBasket is never used to update the state.

Reply:
This will be removed.